createDataset.py

```diff
@@ -4,18 +4,144 @@
 import re
 from datetime import datetime
 
-personName = raw_input('Enter your full name: ')
 fbData = raw_input('Do you have Facebook data to parse through (y/n)?')
 googleData = raw_input('Do you have Google Hangouts data to parse through (y/n)?')
 linkedInData = raw_input('Do you have LinkedIn data to parse through (y/n)?')
 whatsAppData = raw_input('Do you have whatsAppData to parse through (y/n)?')
+discordData = raw_input('Do you have discordData to parse through (y/n)?')
 
 def getWhatsAppData():
```

Comment:
Just wanted to also confirm since the diff is a bit hard to parse. What were the changes to the existing functions? I see the `personName` change is one of them, but do you have a tl;dr of the other ones, if any?

Reply:
All the changes I made, even the smallest ones, should be documented on the `Changelog` above. But yes the only diff to the other functions are, as you noticed, `personName` and also this
> Changed all open(...) commands in createDataset.py to with open(....)
Basically changed all the `openedFile = open(...)` to `with open(...) as openFile:`
It's for good practice, that's all.